src/main/java/frc/robot/subsystems/shooter/ShooterIOReal.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.subsystems.shooter;
+
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.VelocityVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+
+import frc.robot.Constants;
+
+public class ShooterIOReal {
+    
+    private TalonFX bottomShooterMotor = new TalonFX(Constants.BOTTOM_SHOOTER_MOTOR_ID);
```

Comment:
rename to be more concise (can use f2 in vscode to rename all instances of this.)

Reply:
also same feedback about constants